package.json

```diff
@@ -42,6 +42,7 @@
     "license": "gulp checkLicenses",
     "lint": "eslint .",
     "package": "gulp package",
+    "prepare": "gulp buildJavaScriptAndDeps",
```

Comment:
If this is just an alias of `npm run build:deps` could you make it that instead?

Reply:
Your question suggests that you think we should change this to
```
"prepare": "npm run build:deps",
```
and I was going to do that but upon reflection I sort of think that "buildJavaScriptAndDeps" is slightly more clear about what will actually be prepared.  What do you think?

Reply:
If you always want "prepare" to do the exact same thing "build:deps" does, then I think it should be an alias.

If "prepare" actually means something different from "build:deps" it just so happens that right now you want them to do the same thing, then I'm fine with having both commands have the same value.

I just want to make sure that neither command becomes broken cruft, because something gets changed in one place and not the other.

Reply:
> If you always want "prepare" to do the exact same thing "build:deps" does, then I think it should be an alias.
> 
> If "prepare" actually means something different from "build:deps" it just so happens that right now you want them to do the same thing, then I'm fine with having both commands have the same value.

I believe it is the latter.  At the moment, "build:deps" happens to do the things that I think we need to do to prepare to work in the Blockly repository (after installing npm packages, of course).  But this is mostly a coincidence;  in the future we might want `prepare` to do the same thing as `build`, or at least to additionally do `build:langfiles`.

> I just want to make sure that neither command becomes broken cruft, because something gets changed in one place and not the other.

Yes, very sensible.